web_portal_link/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "My Page",
+    "summary": "When cklick the app, system will open my page in new tab.",
```

Comment:
```suggestion
    "name": "Web Portal Link",
```
Our standard practice is make the name consistent with the technical name of the module.  This is to avoid the confusion when searching the module.